tests/MaskedCsrfTokenTest.php

```diff
@@ -36,6 +28,8 @@ private function createCsrfToken(string $token = null): CsrfToken
                 ->method('get')
                 ->willReturn($token);
         }
-        return new CsrfToken($mock);
+        return new MaskedCsrfToken(
```

Comment:
Same here. These tests should, ideally, be ran against both stateful and stateless tokens.

Reply:
Done